src/util.ts

```diff
@@ -195,16 +196,15 @@ export function formatDate(year, month, day) {
 
 /**
  * Split ISO 8601 calendar dates or years into a numerical array.
- * @param {string} date - A date of the format 'YYYY', 'YYYY-MM', or
- *                        'YYYY-MM-DD'.
+ * @param {string} date - A date of the format 'YYYY', 'YYYY-MM', or 'YYYY-MM-DD'.
  * @returns {number[]} Year, month, and day of month respectively.
```

Comment:
Yeah, that's an oversight on my part, I've updated the TS return type but forgot to update the JSDoc type. Fixed as part of 5a519ec9e7e166e97833e9ed987c646da5b73139.

These redundant JSDoc types are annoying me since my first PRs already, I wonder whether they are really required/useful for the JSDoc documentation which we are generating or whether we can get rid of them (and find a way to get ESLint to still accept these as valid JSDoc comments).